p2p/module.go

```diff
@@ -97,18 +88,18 @@ func (*p2pModule) Create(bus modules.Bus) (modules.Module, error) {
 	return m, nil
 }
 
-func (m *p2pModule) SetBus(bus modules.Bus) {
-	// INVESTIGATE: Can the code flow be modified to set the bus here?
-	// m.network.SetBus(m.GetBus())
-	m.bus = bus
-}
+func (m *p2pModule) setupDependencies() {
```

Comment:
it could be anything the module depends on, they are dependencies, currently they are providers, tomorrow who knows.

If you feel strongly about it I can change it